src/olympia/devhub/tasks.py

```diff
@@ -821,6 +821,32 @@ def resize_preview(src, preview_pk, **kw):
         log.error("Error saving preview: %s" % e)
 
 
+@task
+def recreate_previews(addon_pk, **kw):
+    log.info('[1@None] Recreating previews for addon: %s' % addon_pk)
```

Comment:
What is this `[1@None]` prefix for?

Reply:
tbh, no idea.  I lazily copied the log.info from the `resize_preview` method above.

Reply:
https://github.com/mozilla/addons-server/pull/8553/commits/d002da8dc8c33a60929d9ab9fec9643d928c28c2 and it reminded me why that format may be used for test logging and that it wouldn't have worked as a command so https://github.com/mozilla/addons-server/pull/8553/commits/88275247b03aa8b0b7f6070b7ed89da2d3822d1c refactored things.